### ISSUE TITLE
normalizeResponse must return a valid JSON API document... TypeError: Already read

### ISSUE THREAD

**Issue by marisawallace** [davidpett/ember-data-contentful#8]
In `_getContent`:

```
  _getContent(type, params) {
   ...
    }).then((response) => {
      return response.json();
    });
```

The value of `response.json()._result` is:

```
TypeError: Already read
    at consumed (http://localhost:4200/assets/vendor.js:102297:29)
    at Response.Body.text (http://localhost:4200/assets/vendor.js:102389:24)
    at Response.Body.json (http://localhost:4200/assets/vendor.js:102416:19)
    at eval (eval at evaluate (unknown source), <anonymous>:1:10)
    at Object.InjectedScript._evaluateOn (<anonymous>:145:85)
    at Object.InjectedScript._evaluateAndWrap (<anonymous>:137:25)
    at Object.InjectedScript.evaluateOnCallFrame (<anonymous>:151:143)
    at http://localhost:4200/assets/vendor.js:142544:9
    at tryCatch (http://localhost:4200/assets/vendor.js:63920:14)
    at invokeCallback (http://localhost:4200/assets/vendor.js:63935:15)
```

This is causing Ember to throw the "normalizeResponse must return a valid JSON API document" error.

Any idea why this error is happening?

The `fetch` call to Contentful is receiving a 200 back... everything seems fine there. I've tested this in both Firefox and Chrome, and the error is there in both cases.

**Comment by marisawallace** [comment]
This error seems to be fixed by creating a custom serializer:

```
import ContentfulSerializer from 'ember-data-contentful/serializers/contentful';

export default ContentfulSerializer.extend({
});
```

...for each model I'm fetching through Contentful. Still no idea precisely why it started happening in the first place. Things were definitely working for a little while before I added these serializers, so I'm not sure how I introduced the bug / if it's a re-build/caching issue with my local Ember CLI. 

One other note: I mistakenly assumed that the `defaultSerializer` property set on the Contentful Adapter would specify the serializer for the models I'm fetching through Contentful. [This is not the case-- the application serializer still takes precedence.](http://emberjs.com/api/data/classes/DS.Adapter.html#property_defaultSerializer).
